containers/Comments/CommentForm.styles.js

```diff
@@ -7,14 +7,21 @@ export const styles = StyleSheet.create({
     flexDirection: 'row',
     alignItems: 'center',
     paddingHorizontal: SPACINGS.multiplyBy(1),
-    paddingVertical: SPACINGS.multiplyBy(0.75),
+    paddingTop: SPACINGS.multiplyBy(0.75),
+    paddingBottom: SPACINGS.multiplyBy(2)
```

Comment:
I guess this is more of a general thing we need to start doing: have different styles for iOS and Android because this previously looked fine on Android but not iOS. Now it's the other way round.
What we could do in most places is something like 
`paddingBottom: Platform.OS === 'ios' ? SPACINGS.multiplyBy(2) : SPACINGS.multiplyBy(0.75)`
In this particular case it's a bit more complicated I think because as I understand the screenshots in the issue we don't want the big padding if the keyboard is visible, so we have to handle that programatically ?

Reply:
I've looked more into this general issue (I also screwed up the header on iOS) it seems we should probably use the `SafeAreaView` as described [here](https://reactnavigation.org/docs/handling-safe-area/) instead of manually guessing paddings since there could also be weird physical notches etc